pipeline/tasks/gapic_tasks.py

```diff
@@ -61,10 +61,7 @@ def _move_to(self, gapic_config_dir, gapic_api_yaml):
         elif len(gapic_api_yaml) == 0:
             raise ValueError(error_fmt + 'No location specified')
         conf_out = os.path.abspath(gapic_api_yaml[0])
-        if os.path.exists(conf_out):
-            raise ValueError(error_fmt + 'File already exists')
```

Comment:
Everyone has their own way of working. I use Ctrl-R for infrequently used commands in my history, but once they become bread-and-butter, I create aliases for them. They have less cognitive overhead than making sure I have the correct variant of a command from my history.

I'll drop this change from this PR, since I think it's the only controversial bit. I'll add a TODO comment that perhaps it should only be done for local mode, due to the behavior of overwriting an active sandbox.